www/reference/src/components/side-bar/index.js

```diff
@@ -96,7 +100,7 @@ const Sidebar = ({ data, api }) => {
           return <SideBarItem item={s} key={i} />
         })}
       </Flex>
-      <Flex sx={{ py: 4, px: 4, borderTop: "1px solid #efefef" }}>
+      <Flex sx={{ py: 4, px: 4, borderTop: "1px solid var(--theme-ui-colors-separator)" }}>
         <Image src={LogoMuted} alt="Medusa Type" sx={{ height: "10px" }} />
       </Flex>
```

Comment:
comment: I don't believe this is used. And we are also using our old logo here, so either change to the new one or delete `logo-muted.svg` if not used.

Reply:
Since the new logo is used at the top I think it's weird to use it at the bottom again. I'll just delete it.